### PR TITLE
Feature/input styles

### PR DESCRIPTION
**Pull request by Vin-RM**

### CODE REVIEW

**Comment by jkoenig134** [comment]
Pipeline is failing, you shoud update at first the base branch to main then this branch to the base one.

Additionally - please first do some review rounds internally and then assign me.

**Comment by jkoenig134** [comment]
Also - why is this a draft? Typically drafts should not be given out for review.

**Comment by jkoenig134** [comment]
@Vin-RM https://github.com/nmshd/cns-app/pull/90 fixes the bug with the version that cannot be fetched. We have to merge this first.

File: packages/value_renderer/lib/src/widgets/renderers/complex.dart [hunk 1]
```diff
@@ -72,15 +72,12 @@ class _ComplexRendererState extends State<ComplexRenderer> {
     final fieldName = widget.fieldName;
     final translatedText = fieldName.startsWith('i18n://') ? FlutterI18n.translate(context, fieldName.substring(7)) : fieldName;
```
Comment (nicole-eb) [inline packages/value_renderer/lib/src/widgets/renderers/complex.dart:73]: same

**Review by nicole-eb — CHANGES_REQUESTED** [review]

File: packages/value_renderer/lib/src/widgets/inputs/dropdown_select_input.dart [hunk 2]
```diff
@@ -44,16 +46,17 @@ class _DropdownSelectInputState extends State<DropdownSelectInput> {
 
   @override
   Widget build(BuildContext context) {
+    final fieldName = widget.fieldName;
+    final translatedText = fieldName.startsWith('i18n://') ? FlutterI18n.translate(context, fieldName.substring(7)) : fieldName;
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/value_renderer/lib/src/widgets/inputs/dropdown_select_input.dart:50]: I don't think so, fieldName is also used in the `FlutterI18n.translate` call which make the whole line more readable.

**Review by nicole-eb — APPROVED** [review]

File: packages/value_renderer/lib/src/widgets/inputs/segmented_button_input.dart [hunk 3]
```diff
@@ -22,14 +23,14 @@ class SegmentedButtonInput extends FormField<ValueHintsDefaultValue?> {
           onSaved: onSaved,
           validator: validator,
           builder: (FormFieldState<ValueHintsDefaultValue?> field) {
+            final BuildContext context = field.context;
+            final translatedText = fieldName.startsWith('i18n://') ? FlutterI18n.translate(context, fieldName.substring(7)) : fieldName;
```
Comment (jkoenig134) [inline packages/value_renderer/lib/src/widgets/inputs/segmented_button_input.dart:27]: you are using the context only once, so IMO it should not be put in an extra variable.

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**